python/ql/src/Security/CWE-377/InsecureTemporaryFile.ql

```diff
@@ -0,0 +1,20 @@
+/**
+ * @name Insecure temporary file
+ * @description Creating a temporary file using mktemp may be insecure.
```

Comment:
I've rewritten the description, since the query now supports two other insecure functions (but perhaps it's too vague now?). Please let me know if you think it needs further changes.